### PR TITLE
loader: update loader readme and add config example

### PR DESCRIPTION
**Pull request by cwen0**
1. update loader readme 
2. add config example
@qiuyesuifeng @tiancaiamao @iamxy @nolouch  PTAL

### CODE REVIEW

File: loader/README.md [hunk 1]
```diff
@@ -29,10 +28,51 @@ Usage of loader:
...
 ```
 ./bin/loader -d ./test -h 127.0.0.1 -u root -P 4000
 ```
+Or use config file
+```
+./bin/loader -c=config.toml
+```
+
+## Notification
+When loading the file, loader will write the file name which is successful loaded into the checkpoint file. When the loader is broken in the process of loading, we can recovery it according to the checkpoint file and the loader will automatically ignore files that have already been loaded. If you want to reload all the data, remeber to  delete the data that has been loaded and the checkpoint file. The default checkpoint file is `loader.checkpoint`, you can change it by modifying parameter when you start the loader.
```
Comment (qiuyesuifeng) [inline loader/README.md]: When loading the `mydumper` sql file

Comment (qiuyesuifeng) [inline loader/README.md]: When the loader exits unexpectedly

Comment (qiuyesuifeng) [inline loader/README.md]: we can recover

Comment (qiuyesuifeng) [inline loader/README.md]: remember to delete the checkpoint file

Comment (qiuyesuifeng) [inline loader/README.md]: you can change it by specifying the `-checkpoint` flag.

Comment (qiuyesuifeng) [inline loader/README.md]: @queenypingcap PTAL

File: loader/README.md [hunk 2]
```diff
@@ -72,7 +72,7 @@ Or use config file
 ```
 
 ## Notification
-When loading the `mydumper` file, loader will write the file name which is successful loaded into the checkpoint file. When the loader exits unexpectedly in the process of loading, we can recover it according to the checkpoint file and the loader will automatically ignore files that have already been loaded. If you want to reload all the data, remeber to  delete the data that has been loaded and remember to delete the checkpoint file. The default checkpoint file is `loader.checkpoint`, you can change it by specifying the `-checkpoint` flag when you start the loader.
+When loading the `mydumper` sql file, loader will write the file name which is successful loaded into the checkpoint file. When the loader exits unexpectedly in the process of loading, we can recover it according to the checkpoint file and the loader will automatically ignore files that have already been loaded. If you want to reload all the data, remeber to  delete the data that has been loaded and remember to delete the checkpoint file. The default checkpoint file is `loader.checkpoint`, you can change it by specifying the `-checkpoint` flag when you start the loader.
```
Comment (qiuyesuifeng) [inline loader/README.md]: > If you want to reload all the data, remeber to  delete the data that has been loaded and remember to delete the checkpoint file.

Seems we can not to delete the data, only delete the checkpoint file is OK.

File: loader/README.md [hunk 3]
```diff
@@ -1,8 +1,7 @@
 ## loader
```
Comment (queenypingcap) [inline loader/README.md]: Is there a special reason why we name it "loader" instead of "Loader"?

File: loader/README.md [hunk 4]
```diff
@@ -1,8 +1,7 @@
...
+loader is a tool used for multi-threaded restoration of mydumper.
 
 ## How to use
```
Comment (queenypingcap) [inline loader/README.md]: ->Options

File: loader/README.md [hunk 5]
```diff
@@ -1,8 +1,7 @@
...
-
-## Config
 ```
 Usage of loader:
```
Comment (queenypingcap) [inline loader/README.md]: The loader tool has the following available options:

File: loader/README.md (hunk 3 above)
Comment (cwen0) [inline loader/README.md]: This name is consistent with the name of the other tools

File: loader/README.md [hunk 6]
```diff
@@ -29,10 +28,51 @@ Usage of loader:
...
+
+# DB config
+[db]
+host = "127.0.0.1"
+user = "root"
+password = ""
+port = 4000
+```
+
 ## Example
```
Comment (queenypingcap) [inline loader/README.md]: You can run the following commands:

File: loader/README.md [hunk 7]
```diff
@@ -29,10 +28,51 @@ Usage of loader:
...
 ```
 ./bin/loader -d ./test -h 127.0.0.1 -u root -P 4000
 ```
+Or use config file
```
Comment (queenypingcap) [inline loader/README.md]: Or use the "config.toml" file:

File: loader/README.md [hunk 8]
```diff
@@ -29,10 +28,51 @@ Usage of loader:
...
+```
+./bin/loader -c=config.toml
+```
+
+## Notification
+When loading the `mydumper` sql file, loader will write the file name which is successful loaded into the checkpoint file. When the loader exits unexpectedly in the process of loading, we can recover it according to the checkpoint file and the loader will automatically ignore files that have been loaded. The default checkpoint file is `loader.checkpoint`, you can change it by specifying the `-checkpoint` flag when you start the loader.
```
Comment (queenypingcap) [inline loader/README.md]: When loading the `mydumper` SQL file, the loader tool writes the file names which are successfully loaded into the checkpoint file. If the loader exits unexpectedly in the process of loading, we can recover it according to the checkpoint file and the loader will automatically ignore the files that have been loaded. The default checkpoint file is `loader.checkpoint`, you can change it by specifying the `-checkpoint` flag when you start the loader.

File: loader/README.md [hunk 9]
```diff
@@ -63,16 +63,17 @@ port = 4000
...
 ```
-Or use config file
+Or use the "config.toml" file:
 ```
 ./bin/loader -c=config.toml
 ```
 
 ## Notification
-When loading the `mydumper` sql file, loader will write the file name which is successful loaded into the checkpoint file. When the loader exits unexpectedly in the process of loading, we can recover it according to the checkpoint file and the loader will automatically ignore files that have been loaded. The default checkpoint file is `loader.checkpoint`, you can change it by specifying the `-checkpoint` flag when you start the loader.
+When loading the `mydumper` SQL file, the loader writes the file names which are successful loaded into the checkpoint file. If the loader exits unexpectedly in the process of loading, we can recover it according to the checkpoint file and the loader will automatically ignore the files that have been loaded. The default checkpoint file is `loader.checkpoint`, you can change it by specifying the `-checkpoint` flag when you start the loader.
```
Comment (queenypingcap) [inline loader/README.md]: successful-> successfully

File: loader/README.md [hunk 10]
```diff
@@ -1,9 +1,9 @@
-## loader
-loader is a tool used for multi-threaded restoration of mydumper.
+## Loader
```
Comment (queenypingcap) [inline loader/README.md]: If the name is changed to "Loader", please change it in the whole README to make it consistent.

**Comment by queenypingcap** [comment]
LGTM

**Comment by qiuyesuifeng** [comment]
LGTM

**Pull request merged**